projects/1.20/assets/1UNKNOWN/appendhextome/lang/zh_cn.json

```
@@ -0,0 +1,11 @@
+{
+	"death.attack.appendicitis": "%s的阑尾涨裂了",
+	"death.attack.appendicitisInstant": "%s的阑尾不足以容纳%s的灵魂",
+	"appendhextome.entry.appendix": "阑尾附录",
+	"appendhextome.page.readwrite.1": "我在我的身体里发现了一个奇怪的器官；从它的位置看来，它可能会参与到消化过程中。我称其为“阑尾”，因为它只是附在大肠末端的一个小挂件……后页的图案可向其读写任意 iota，但仍要多加小心。要是生物体内存了过多的 iota 信息，阑尾大概会出现严重的炎症。甚至可能致死。",
+	"appendhextome.page.readwrite.2": "从我的考察看来，阑尾中 iota 的大小和深度的乘积大于 20 时会开始出现炎症症状，每次跨越此界限时炎症会加深。由于该 iota 存储位置就在我的体内，我认为只有当这幅躯体被损坏到无法修复的时候，其中的 iota 才会消失。",

```

```suggestion
	"appendhextome.page.readwrite.2": "从我的考察看来，阑尾中 iota 的大小和深度的乘积大于 20 时会开始出现炎症症状，每次跨越此界限时炎症会加深。由于该 iota 存储位置就在我的体内，我认为当这幅躯体被损坏到无法修复的时候，其中的 iota 就会随之消失。",
```
感觉原文（`if this body ... beyond repair, the iota stored within would be destroyed as well.`）没有限定的意思吧，~~虽说这阑尾好像真是个不可擦的神秘东西~~